deploy/cpd-image-build/version.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "1.78.0"
+      version = "1.78.2"
```

Comment:
this is a module right? So it should use `>=1.78.2, <2.0.0` - please make sure all the modules are using this format. The DA code itself should lock to a version. Module use range